source/sass/patterns/molecules/tag-list.scss

```diff
@@ -1,9 +1,18 @@
 @import "../../mixins/sizes";
+@import "../../mixins/spacing";
 @import "../../mixins/typography";
 @import "../../variables/baselinegrid";
+@import "../../variables/contentHeader";
+@import "../../variables/grid";
 
 .tag-list {
```

Comment:
`tag-list` pattern styles need some work, but that's out of scope for this PR.

Reply:
Still relevant?